### DISCUSSION TITLE
How long is the average honey-bee stinger?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#383] [Galaxy Trivia] [answered]
*give answer in micrometers μm*

**Comment by peterbaccaro** [comment]
The average honey-bee stinger is **1.62 mm** in length.

**Comment by thuongtruong1009** [comment]
The average honey-bee stinger is approximately 150-200 micrometers (μm) in length.

**Comment by ParanoidUser** [comment]
**Hint 2:** 1 mm = 1000 μm

**Comment by Roberto-Larivera** [comment, marked as answer]
C'è una differenza dì pungiglione tra un'ape operaia e una regina.

- La lunghezza media del pungiglione di un'ape operaia è di circa 1500-3000 μm.
- La lunghezza del pungiglione di un'ape regina può raggiungere i 6000 μm.

**Reply by ParanoidUser** [reply]
Ben fatto! 👏👏 Congratulazioni per il tuo meritato [x2 Galaxy Brain 🥉](https://github.com/Roberto-Larivera?tab=achievements&achievement=galaxy-brain) badge. Continua per sbloccare i livelli successivi del distintivo (Argento per 16 risposte), e non esitare a postare le tue domande nella community. 😊🎉

**Reply by Roberto-Larivera** [reply]
Grazie mille è divertente rispondere ai quiz
